domains/certificates/Query.API/API/Query.API/Controllers/TransferCertificateController.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using API.Query.API.ApiModels.Requests;
+using API.Query.API.ApiModels.Responses;
+using API.TransferCertificateService;
+using Baseline;
+using CertificateEvents;
+using MassTransit;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Query.API.Controllers;
+
+[Authorize]
+[ApiController]
+public class TransferCertificateController : ControllerBase
+{
+    [HttpPost]
+    [ProducesResponseType(200)]
+    [Route("api/certificates/production/transfer")]
+    public async Task<IActionResult> TransferCertificate(
+        [FromBody] TransferCertificate transferCertificate,
+        [FromServices] ITransferCertificateService service
```

Comment:
As far as I remember, it is possible to get IRequestClient from the service collection:

```suggestion
        [FromServices] IRequestClient<TransferCertificate> requestClient
```